resources/user-agents/crawlers/general-crawlers/general-crawlers.json

```diff
@@ -2328,6 +2328,54 @@
         {
           "match":  "NinjBot/*",
           "browser":  "ninj-bot"
+        },
+        {
+          "match":  "Yewno-Open",
+          "browser":  "yewno-open"
+        },
+        {
+          "match":  "M",
+          "browser":  "m"
+        },
+        {
+          "match":  "deepcrawl",
+          "browser":  "deepcrawl"
+        },
+        {
+          "match":  "Magnetbot",
+          "browser":  "magnetbot"
+        },
+        {
+          "match":  "Caliperbot/1.0 (+http://www.conductor.com/caliperbot)",
+          "browser":  "caliperbot"
+        },
+        {
+          "match":  "ISP_TESTING_SUITE",
+          "browser":  "isp_testing_suite"
+        },
+        {
+          "match":  "Asana/1.4.0 WebsiteMetadataRetriever",
```

Comment:
Please replace the version numer with a `*`

Reply:
Replaced